crates/iota-rosetta/src/main.rs

```diff
@@ -208,37 +198,36 @@ async fn wait_for_iota_client(rpc_address: String) -> IotaClient {
 /// This method reads the keypairs from the IOTA keystore to create the
 /// PrefundedAccount objects, PrefundedAccount will be written to the
 /// rosetta-cli config file for testing.
-fn read_prefunded_account(path: &Path) -> Result<Vec<PrefundedAccount>, anyhow::Error> {
-    let reader = BufReader::new(File::open(path).unwrap());
-    let kp_strings: Vec<String> = serde_json::from_reader(reader).unwrap();
-    let keys = kp_strings
-        .iter()
-        .map(|kpstr| {
-            let key = IotaKeyPair::decode(kpstr);
-            key.map(|k| (IotaAddress::from(&k.public()), k))
-        })
-        .collect::<Result<BTreeMap<_, _>, _>>()
-        .unwrap();
+fn read_prefunded_account(path: &PathBuf) -> Result<Vec<PrefundedAccount>, anyhow::Error> {
+    let key_store =
+        FileBasedKeystore::new(path).map_err(|e| anyhow!("Failed to open keystore file: {}", e))?;
```

Comment:
```suggestion
        FileBasedKeystore::new(path).map_err(|e| anyhow!("Failed to open keystore file: {e}"))?;
```

Reply:
Done in https://github.com/iotaledger/iota/pull/7704/commits/db926050524a96d631e0116ba1f39d72159cd22c